Project/src/MakeCall/CallCard.js

```diff
@@ -797,9 +797,12 @@ export default class CallCard extends React.Component {
                                                 />
                                             }
                                             <div>
-                                            <h3>Volume Visualizer</h3>
+                                            {
+                                                (this.state.callState === 'Connected') && !this.state.micMuted && !this.state.incomingAudioMuted &&
+                                                <h3>Volume Visualizer</h3>
+                                            }
                                             {   
-                                                (this.state.callState === 'Connected') && !this.state.micMuted &&
+                                                (this.state.callState === 'Connected') && !this.state.micMuted && !this.state.incomingAudioMuted &&
```

Comment:
in this case if remote or local audio is muted both volume visualizers will be hidden? is that the expected behavior?

Reply:
yes